src/command_modules/azure-cli-extension/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.0.9
+++++++
+Added support for a hidden -extra-index-url to extension add and update commands.
```

Comment:
I would remove 'hidden' from this message. Having it hidden but calling out the hiddenness in the release notes is slightly ironic.
Also, it should be `--extra-index-url` rather than `-extra-index-url`.